README.md

```diff
@@ -23,11 +25,39 @@ var options = {
   sessionIndex: '_faed468a-15a0-4668-aed6-3d9c478cc8fa'
 };
 
-var signedAssertion = saml.create(options);
+saml.create(options, (err, samlAssertion) => {
```

Comment:
Synchronous invocation is actually supported unless the `encryptionCert` option has been passed in, at which point `callback` becomes required. It's a bit silly but that's how the library has behaved.

https://github.com/auth0/node-saml/blob/06acc0238d7161c123f2f6924aa9f5984a5a2f32/lib/saml20.js#L261-L263

I'm ok with the change though.